### PR TITLE
58 - Refresh README.md; add INFRA.md, DEV.MD, RULES.md; remove rules.txt

### PR DESCRIPTION
**Pull request by tbitonti**
Refresh of top level README files.

This is a first draft which is provided for review.  Many details, such as exact file names and formatting, are in motion.

### CODE REVIEW

File: DEV.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+  * File update actions;
+  * Which are property file driven;
+  * Composed with container actions;
+  * And invoked using java and command line APIs.
+
+Further:
+
+  * The core operation is to create a transformed copy of a root input file:
+    ** The core inputs are a target root input file, a target root output file, and a set of processing options.
```
Comment (bjhargrave) [inline DEV.md:13]: For nested bullets you need 4 spaces indent from the outer level bullet.

File: DEV.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
++ Top Level Design
+
+The Eclipse Transformer comprises:
+
+  * File update actions;
```
Comment (bjhargrave) [inline DEV.md:5]: The top level bullets should not have leading spaces. Just put the `*` at the start of the line.

File: DEV.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
++ Top Level Design
```
Comment (bjhargrave) [inline DEV.md:1]: I think everywhere you used `+`, you need to use `#`.

File: DEV.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+* Class org.eclipse.transformer.Transformer" provides a standard "main" command line entry point.
+
+* Class org.eclipse.transformer.Transformer" also provides for direct invocation from java using
+  "int Transformer.runWith(PrintStream, PrintStream, String[])".
+
+A third API is provided for Jakarta type transformations:
+
+* Class org.eclipse.transformer.jakarta.JakartaTransformer" provides an addition "main" command line entry point.  "JakartaTransformer" is packaged with default update properties files and supplies these as default property files for the several update properties files.
+
+package org.eclipse.transformer;
```
Comment (bjhargrave) [inline DEV.md:59]: Use a fenced code block around the code https://github.github.com/gfm/#fenced-code-blocks

~~~markdown
```java
javacode
```
~~~

File: DEV.md [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+        if ( rc == SUCCESS_RC ) {
+            System.out.println("Return Code [ 0 ]: Success");
+        } else {
+            System.err.println("Return Code [ " + rc + " ]: Failure [ " + RC_DESCRIPTIONS[rc] + " ]");
+        }
+        return rc;
+    }
+}
+
+--
```
Comment (bjhargrave) [inline DEV.md:112]: You need 3 for a horizontal rule.

File: README.md [hunk 6]
```diff
@@ -1,50 +1,115 @@
-Liberty Jakarta Prototyping Environment
++ Eclipse Transformer
```
[the remark this answers is not in the document]
Comment (tbitonti) [reply README.md]: The markdown was not correct.  I've refreshed all of the files and verified that the markdown is now correct.

File: DEV.md (hunk 1 above)
Comment (tbitonti) [reply DEV.md]: Corrected by refresh of the files.

File: DEV.md (hunk 2 above)
Comment (tbitonti) [reply DEV.md]: Corrected by refresh of the files.

File: DEV.md (hunk 4 above)
Comment (tbitonti) [reply DEV.md:61]: Corrected by refresh of the files.

File: DEV.md (hunk 5 above)
Comment (tbitonti) [reply DEV.md]: Corrected by refresh of the files.

**Pull request merged**